packages/app/background-jobs-common/src/background-job-processor/errors/MutedUnrecoverableError.ts

```diff
@@ -3,9 +3,12 @@ import { UnrecoverableError } from 'bullmq'
 export const MUTED_UNRECOVERABLE_ERROR_SYMBOL = Symbol.for('MUTED_UNRECOVERABLE_ERROR_KEY')
 
 export class MutedUnrecoverableError extends UnrecoverableError {
-  constructor(message?: string) {
+  public readonly details?: Record<string, unknown>
+
+  constructor(message?: string, details?: Record<string, unknown>) {
     super(message)
     this.name = UnrecoverableError.name
+    this.details = details
```

Comment:
I wonder if we should have both details (for general usage) and cause (for source errors) fields, as cause, being an instance of an error, needs to be serialized differently.

but we can test with just details first and see if cause from that would get logged correctly